src/attack.rs

```diff
@@ -108,6 +110,16 @@ fn activate_hitbox(
                 ));
             }
         }
+        if let Ok((animation, explodable)) = explodable_query.get(**parent) {
+            if animation.current_frame >= attack_frames.startup
+                && animation.current_frame <= attack_frames.active
+            {
+                commands.entity(entity).insert(Collider::cuboid(
+                    explodable.attack.hitbox.size.x / 2.,
+                    explodable.attack.hitbox.size.y / 2.,
+                ));
+            }
+        }
```

Comment:
I'd like to avoid special casing this activation code, I've done a bit of a refactor, added the hitbox metadata directly to `Attack` now which should let you remove this extra query and block if you add a `AvailableAttacks` component to the parent entity of the attack (im guessing the bomb entity in this case).